src/utils/groupNumbers.ts

```diff
@@ -0,0 +1,26 @@
+export const groupNumbers = (numbers: number[]): number[][] =>
```

Comment:
For clarity this could be renamed to `groupConsecutiveNumbers`

Reply:
Donets